lib/email_api/email_api.py

```diff
@@ -0,0 +1,219 @@
+import os
+import random
+from smtplib import SMTP_SSL
+from email.header import Header
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email.utils import formatdate
+from typing import Dict, List
+
+
+class EmailApi:
```

Comment:
Nit:

To make testing this easier, I was wondering if it's worth moving the pack config to `__init__` as a member (i.e. `pack_config`), as I want to port the Jupyter stuff to use the same pattern too?

It would be nice to come up with an established pattern to get keys / creds code-wise between us that works across all of them

Reply:
Edit: I think this worth a discussion on call, as it would be nice to either adopt the AWS pack pattern or pass through the pack config entirely through